CHANGELOG.md

```diff
@@ -2,14 +2,56 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x reached End of Life on September 8th, 2021.** See the changelog entry [here](https://www.twilio.com/changelog/end-of-life-complete-for-unsupported-versions-of-the-programmable-video-sdk). Support for the 1.x version ended on December 4th, 2020.
 
+2.23.0 (In Progress)
+====================
+New Features
+------------
+
+- This release adds private beta support for 3rd party noise cancellation solution. You need to host twilio approved 3rd party plugin on your web server to enable noise cancellation. Please fill out [this form](https://forms.gle/eeFyoGJj1mgMrxN88) to request access to the 3rd party plugin.
```

Comment:
This is called private beta because customer need to request access, even though the release is public. Is this accurate?

Reply:
yeah, that does seem bit odd, but I think its best way forward - CC: @anthonylazaro